src/scm/Model.ts

```diff
@@ -296,7 +296,7 @@ export class Model implements Disposable {
             const uri = Uri.file(clientFile);
             const resource: Resource = new Resource(uri, change, action);
 
-            if (change === 'default change') {
+            if (change.startsWith('default')) {
```

Comment:
I noticed that source depot actually outputs "... change default" with fstat (without ztag). If perforce really is outputting "default change", even without ztag, this should still catch it, but please make sure files in the default change show up.